RxExample/RxExample/Examples/WikipediaImageSearch/Views/WikipediaSearchViewController.swift

```diff
@@ -13,15 +13,10 @@ import RxCocoa
 #endif
 
 class WikipediaSearchViewController: ViewController {
+    private let searchController = UISearchController(searchResultsController: UITableViewController())
+    
+    @IBOutlet weak var searchBarContainer: UIView!
     
-    private var resultsTableView: UITableView {
```

Comment:
If we add some variable for having more convenience to access properties, when the class grows up, next person who needs to maintain the project probably doesn't know there was some variables can be use.
So I prefer to eliminate adding some variables like this.

However, indeeds that is not the thing which must be done and there are exist code in here, I should follow the codes to minimum changes(If I wants to put these parts back, should I rebase? or just add some commits).

And after using UISearchController instead, SearchResultController had been setup as UISearchController constructed.
At beginning of `configureResultController ` method, it just ensure SearchResultController is UITableViewColler.
Then there must be a tableView below UITableViewController so I used the `!`.